test/ui-test/testSuites/suite_communities/shared/steps/communitySteps.py

```diff
@@ -266,7 +266,11 @@ def step(context, option:str, list):
 
 @Then("\"|any|\" button is present")
 def step (context, action_button_name):
-    _statusCommunityScreen.verify_action_button_enabled(action_button_name)   
+    _statusCommunityScreen.verify_action_button_enabled(action_button_name)
+      
+@Then("user is able to open Back up modal and copy private key")
```

Comment:
I think we agreed on using `the` before (@glitchminer ?? ):
```suggestion
@Then("the user is able to open Back up modal and copy private key")
```